### PR TITLE
More bugfixes

### LINKED ISSUE #4532: If a Mech gets Ioned, it will switch into maintenance mode and cannot be switched out of maintenance mode.
Yet another anecdotal report from one of my friends, they got Ioned by Security whilst in a tunnel and just couldn't get the Mech out of Maintenance mode.

### LINKED ISSUE #4847: You cannot eject the contents of a microwave if it is unpowered
You should really be able to take stuff out of a microwave even if it doesn't have electricity flowing. It's a mechanical latch.

### LINKED ISSUE #4986: Cyborg Laser Rifle Glitch
Problem; When a borg is dead and ghosted if it comes back it's laser rifle or any weapons that has charges will not charge anymore
Reproduce; 1. get fucked as borg (combat module or emagged sec module)
2. get revived
3. see how you cant get any more charges in weapons

Game ID; bVh-dlkL
Admin; ornias

### LINKED ISSUE #5009: Surface Level Pool doors Labled holodeck
The doors near surface cryo and the pool are labeled holodeck

### PR DESCRIPTION
**Pull request by Sindorman**
- Microwave can now eject items that are inside even when there is no power to it using `Eject content` verb. Fixes #4847 

- Mechs who were IONed and got into maint mode(while have maint mode change forbidden) can now change if they are allowed  to switch maint mode by DNA owner. Fixes #4532

- Changed door names. Fixes #5009 

- Found important bug in `rejuvinate` proc of living.dm It would crash when called on mobs that do not contain any reagents in it (AI, borgs, some simple animals). Effectively fixes #4986 

- Renamed sprite for basic service module so their eyes overlay is not missing anymore.

### CODE REVIEW

**Comment by Sindorman** [comment]
!review

File: code/game/machinery/kitchen/microwave.dm [hunk 1]
```diff
@@ -410,6 +410,11 @@
 	return ffuu
 
 /obj/machinery/microwave/Topic(href, href_list)
+	if(stat & BROKEN)
+		return
+	if(href_list["action"] == "dispose")
```
Comment (skull132) [inline code/game/machinery/kitchen/microwave.dm]: This should be in the switch at line 421.

And all of these should be after the `if (..())` check. Which is usually a usability and security call.

File: html/changelogs/Sindorman-bugfixes.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+author: PoZe
+
+delete-after: True
+
+changes:
+  - bugfix: "Mechs that were IONed and got into maintenance mode while having it forbidden to switch the mode can now be unlocked by their DNA owners. By being able to allow maintenance mode."
```
Comment (skull132) [inline html/changelogs/Sindorman-bugfixes.yml]: Ion is not an acronym. Lowercase the word `ioned`. Might want to replace with `that were shot with an ion gun`.

File: code/game/machinery/kitchen/microwave.dm (hunk 1 above)
Comment (Arrow768) [reply code/game/machinery/kitchen/microwave.dm]: Thats is the problem.
the ..() can not be before the action check. As it also checks for no power.

`/obj/machinery/CanUseTopic(var/mob/user)
	if(stat & BROKEN)
		return STATUS_CLOSE

	if(!interact_offline && (stat & NOPOWER))
		return STATUS_CLOSE

	return ..()`

Comment (Sindorman) [reply code/game/machinery/kitchen/microwave.dm]: Yeah, me and Arrow couldn't come up with code that will have it in a switch. as `..()` will just quit proc before switch if no power

**Comment by skull132** [comment]
In dev chat we thought of a simple fix for this, @Sindorman . Why the heck is this even an UI option? The UI is meant to represent, well, the interface of the machine. Cleaning should be a right click action/a verb.

Or alternatively, trying to open it with the UI when no power is available should send a message to you about it being out of power and that's it.

**Comment by Sindorman** [comment]
@skull132 Changed!

**Pull request merged**